ui/components/multichain/pages/connections/components/site-not-connected.stories.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { SiteNotConnected } from './site-not-connected';
+
+export default {
+  title: 'Components/Multichain/SiteNotConnected',
+};
+export const DefaultStory = () => <SiteNotConnected />;
+
+DefaultStory.storyName = 'Default';
```

Comment:
Seems like this story is broken

![storybroken](https://github.com/MetaMask/metamask-extension/assets/8112138/abe0f124-f0b3-4013-be7a-9d61f8a4a8f3)